impl/e2e.schema.json

```diff
@@ -83,7 +93,8 @@
                       {
                         "type": "array",
                         "items": {
-                          "type": "integer"
+                          "type": "integer",
+                          "minimum": 0
```

Comment:
Do you really have to specify a minimum of zero?  I thought that was implied.

Reply:
For the length of a list, or is this a per-value constraint?  I'm still a JSON Schema neophyte.